home/export_content_pages.py

```diff
@@ -80,6 +81,7 @@ class ExportRow:
     media_link: str = ""
     related_pages: str = ""
     footer: str = ""
+    language_code: str = ""
```

Comment:
I think we should also remove the `locale` from the export, so that we have just a single column referencing the language.